tests/setup.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="ccftools",
```

Comment:
Name is very much tentative for now. I don't think that `ccf` is ideal (users of the framework may think they can run a network, etc. from Python); `ccfclients` might be too restrictive (we will probably end up release more utilities in there).

Reply:
I think `ccf` would be alright, it's understood that the Python package is going to be supporting tooling/clients, see k8s, zookeeper, etcd.

Reply:
`ccf` it is